src/database/sql/audio.py

```diff
@@ -500,7 +536,8 @@ def get_loudness_range(self, audio_id: str) -> str:
         query = """
             SELECT loudness_range, last_updated
             FROM Audio
-            WHERE audio_release=:rel AND audio_side=:side AND audio_track=:track
+            WHERE audio_release=:rel AND audio_side=:side
+            AND audio_track=:track
```

Comment:
If you want this AND `audio_track=:track` to be on a new line, please indent it.